src/Toml/Bi/Combinators.hs

```diff
@@ -169,18 +169,13 @@ arrayOf bimap key = Codec input output
         a <$ modify (\(TOML vals tables) -> TOML (HashMap.insert key anyVal vals) tables)
 
 -- | Bidirectional converter for @Maybe smth@ values.
-maybeT :: forall a . (Key -> TomlCodec a) -> Key -> TomlCodec (Maybe a)
-maybeT converter key = let codec = converter key in Codec
-    { codecRead  = (Just <$> codecRead codec) `catchError` handleNotFound
+diOptional :: (Functor r, Functor w, Applicative w) => Codec r w c a -> Codec r w (Maybe c) (Maybe a)
+diOptional Codec {..} = Codec
+    { codecRead = Just <$> codecRead
     , codecWrite = \case
         Nothing -> pure Nothing
-        Just v  -> codecWrite codec v >> pure (Just v)
+        Just c -> Just <$> codecWrite c
```

Comment:
Nice! You've just implemented `Traversable` instance for `Maybe`. If you look at this instance, you probably might notice how to implement `codecWrite` in more concise way:

* https://hackage.haskell.org/package/base-4.12.0.0/docs/src/Data.Traversable.html#line-232

Reply:
If so, maybe someday I will implement somewhat function for `relude` :)